src/Lang/en/alert.php

```diff
@@ -4,11 +4,11 @@
     'success' => [
         'title'  => 'Well done!',
         'reason' => [
-            'submitted_to_post'       => 'Response successfully submitted to discussion.',
-            'updated_post'            => 'Successfully updated the Discussion.',
-            'destroy_post'            => 'Successfully deleted the response and discussion.',
-            'destroy_from_discussion' => 'Successfully deleted the response from the discussion.',
-            'created_discussion'      => 'Successfully created a new discussion.',
+            'submitted_to_post'       => 'Response successfully submitted to '.strtolower(Config::get('chatter.titles.discussion')).'.',
```

Comment:
I would not go for using the config on the title for discussion since then you cannot translate the title to other languages as it will always follow the route slug.

Reply:
the idea of langs are files that users doesn't need to touch, but to solve this, either langs are taken out from vendors folder and copied to the resources lang folder and from there they can modify what they want, or if you have another idea, that would be welcome